test/test/manifests/site.pp

```diff
@@ -1,54 +1,55 @@
 Exec {
-  path => [ "/bin/", "/sbin/" , "/usr/bin/", "/usr/sbin/", "/usr/local/bin" ]
+  path => [ '/bin/', '/sbin/' , '/usr/bin/', '/usr/sbin/', '/usr/local/bin' ]
 }
 
 case $operatingsystem {
-  'RedHat', 'CentOS': { 
+  'RedHat', 'CentOS': {
     package { 'rubygems':
-        name => 'rubygems',
         ensure => present,
+        name   => 'rubygems',
     }
 
     case $::operatingsystemrelease {
       /^5.*/: {
         package { 'epel5':
-          source => 'http://mirror.vcu.edu/pub/gnu_linux/epel/5/i386/epel-release-5-4.noarch.rpm',
           ensure => present,
+          source => 'http://mirror.vcu.edu/pub/gnu_linux/epel/5/i386/epel-release-5-4.noarch.rpm',
```

Comment:
It seems that this change causes the `epel5` not to be installed in the test environment and thus the tests fail :( not sure what the best way to fix this would be that also makes the linter happy.